client/src/Chat.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+ 
+import {
+    ApolloClient,
+    InMemoryCache,
+    ApolloProvider
+  } from "@apollo/client";
+
+  const client = new ApolloClient({
+    uri: 'http://localhost:4000/',
```

Comment:
Are we ok with uri value hardcoded? If yes for now and it will be fixed in future, then no problems.